MarkBind/markbind
# Dependency update to remove security vulnerabilities

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Dependency update



**What is the rationale for this request?**

Two security vulnerabilities for npm packages used by MarkBind appeared recently. This PR fixes them.

**What changes did you make? (Give an overview)**

See propesed commit message.

**Testing instructions:**

1. `npm i` does not change any files
2. `npm audit` reveals no security vulnerabilities
3. `npm run test` should print out the same deprecation warnings (introduced in #1057) as version 2.12.0

**Proposed commit message: (wrap lines at 72 characters)**

Update to remove security vulnerabilities

jest updated from 24.9.0 to 25.1.0.
(https://www.npmjs.com/advisories/1488)
kind-of updated from 6.0.2 to 6.0.3.
(https://www.npmjs.com/advisories/1490)
